### DISCUSSION TITLE
Ask Fanuc Driver for Doosan machine

### DISCUSSION THREAD

**Discussion by seryu92** [Ladder99/fanuc-driver#64] [Q&A]
Hello,

i am from indonesia, right now i am working with my thesis for college.
i am also works at manufacturing company, we have lot of Doosan Machine with fanuc controller.

i'm planning to make digitalization for all the CNC Machine, so we can have much structured data.

i want to retrieve data from the CNC machine to PC Server (HP Proliant ML10 G9), can you guide me where should i start?

Thank you

**Comment by MRIIOT** [comment]
Watching the videos linked in the releases would be a good start.

https://github.com/Ladder99/fanuc-driver/releases

**Reply by seryu92** [reply]
already working to collect 2 machine data, in local pc server.
Now still searching a way to send the data to mysql shared hosting.

**Reply by MRIIOT** [reply]
You will want to check out the Ladder99 repository.  Doosan -> MTConnect -> Adapter to MySql.

**Reply by seryu92** [reply]
is docker necessary?
i can't install it since the pc using windows server 2016.

**Reply by MRIIOT** [reply]
It is not.
